### DISCUSSION TITLE
朋友们，撞鬼了！！！

### DISCUSSION THREAD

**Discussion by ran145s** [share121/inter-knot#1185] [其他]
我是住在_十三分街_的，昨天下午喝了点酒，回家走半路一不小心睡着了，在马路上睡了几个小时……等我醒来的时候，那个天已经黑透了，云朦朦胧胧的把月亮遮了一半，看不清楚什么东西，更别提我还正好把眼镜摔坏了……
当时我就想，这还能怎么着？摸着回家呗！
于是我就那么走着回去，醉酒刚刚醒来，头晕乎乎的，一不小心还走错路了！额，我平时也没这么马虎，纯粹就是喝多了！对，都是酒精的错！
诶，反正就那么个意思……我走错路了，到了一个陌生的地方，至少这不是我们十三分街。我一寻思，这完啦，今晚算是回不了家了！不过咱也没有对象，也没人等着咱回去过七夕就是了，呵呵呵……
也就是那个时候，我撞见了个可怕的玩意儿！前面说过了，当时吧，那个天很暗，咱没眼镜看不清东西，我一开始只看见了一大坨白色的东西，站在墙角，我好奇的往过凑了一下，那东西好像还有鼻子有眼的！更可怕的是，它的蛋蛋好像长在了脸上！还没等我弄清楚它到底是个什么玩意，它突然发出了一声怪叫——
那声音好像在说人话！我知道这很荒谬，但我当时太害怕了，也没怎么听清楚，只记得它说的好像是什么“仄瑞四名，里也要为”什么什么的。
哎，希望有人能告诉我这是什么东西，我现在都不敢睡觉的。
![f6e72fc7e7844f5b927c7f06d155068](https://github.com/user-attachments/assets/52339946-219d-469d-a628-04a1c61629ae)

**Comment by Pasukalu** [comment]
其实这是七夕的天降对象
